src/cloud-code/Parse.Cloud.js

```diff
@@ -349,6 +349,49 @@ ParseCloud.afterLogout = function (handler) {
   triggers.addTrigger(triggers.Types.afterLogout, className, handler, Parse.applicationId);
 };
 
+/**
+ *
+ * Registers the before password reset request function.
+ *
+ * **Available in Cloud Code only.**
+ *
+ * This function provides control in validating a password reset request
+ * before the reset email is sent. It is triggered after the user is found
+ * by email, but before the reset token is generated and the email is sent.
+ *
+ * ```
+ * Parse.Cloud.beforePasswordResetRequest((request) => {
+ *   // Validate email or user properties
+ *   if (!request.object.get('emailVerified')) {
+ *     throw new Parse.Error(Parse.Error.EMAIL_NOT_FOUND, 'Email not verified');
+ *   }
+ * })
+ *
+ * ```
+ *
+ * @method beforePasswordResetRequest
+ * @name Parse.Cloud.beforePasswordResetRequest
+ * @param {Function} func The function to run before a password reset request. This function can be async and should take one parameter a {@link Parse.Cloud.TriggerRequest};
```

Comment:
```suggestion
 * Registers the before password reset request function.
 *
 * **Available in Cloud Code only.**
 *
 * This function provides control in validating a password reset request
 * before the reset email is sent. It is triggered after the user is found
 * by email, but before the reset token is generated and the email is sent.
 *
 * Code example:
 *
 * ```
 * Parse.Cloud.beforePasswordResetRequest(request => {
 *   if (request.object.get('banned')) {
 *     throw new Parse.Error(Parse.Error.EMAIL_NOT_FOUND, 'User is banned.');
 *   }
 * });
 *
 * ```
 *
 * @method beforePasswordResetRequest
 * @name Parse.Cloud.beforePasswordResetRequest
 * @param {Function} func The function to run before a password reset request. This function can be async and should take one parameter a {@link Parse.Cloud.TriggerRequest};
```